packager/packager.cc

```diff
@@ -598,6 +598,39 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
                 return core::packages::StrictDependenciesLevel::Dag;
         }
     }
+
+    bool importsTransitively(const core::GlobalState &gs, const PackageInfo &otherPkg) const {
```

Comment:
Minor: the implementation only needs a `MangledName` from `otherPkg` in this. It might be worth to lock this down in the signature.

I point this out because soon this will be akin to taking a `core::ClassOrModule` instead of a `core::ClassOrModuleRef`